app/controllers/contact/govuk/accessible_format_requests_controller.rb

```diff
@@ -82,4 +102,28 @@ def display_optional_question?
     trigger_value = current_question[:"trigger-value"]
     params[trigger_key] == trigger_value
   end
+
+  def content_item
+    @content_item ||= GdsApi.publishing_api.get_content(params[:content_id]).to_h
+  end
+
+  def content_base_path
+    content_item["base_path"]
+  end
+
+  def content_attachments
+    content_item.dig("details", "attachments")
+  end
+
+  def requested_attachment
+    @requested_attachment ||= content_attachments.find { |a| a["id"] == params[:attachment_id] }
```

Comment:
This one doesn't seem to work, I think it might be that `find ` by implicit id is available on ActiveRecord models - but `content_attachments` is simply an array of hashes?

Reply:
Oops, my mistake!